src/Centreon/Infrastructure/PlatformInformation/PlatformInformationRepositoryRDB.php

```diff
@@ -80,4 +91,82 @@ public function findPlatformInformation(): ?PlatformInformation
 
         return $platformInformation;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function updatePlatformInformation(PlatformInformation $platformInformation): void
+    {
+        try {
+            $this->db->beginTransaction();
+            $deleteQuery = $this->db->prepare(
+                $this->translateDbName(
+                    "DELETE FROM `:db`.informations WHERE `key` <> 'version' AND `key` <> 'appKey'"
```

Comment:
If any data is inserted without being referenced as a property of the PlatformInformation class it will be automatically deleted. Too risky.